websites/site-template/gatsby-config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  pathPrefix: '/site-template',
+  siteMetadata: {
+    title: 'CHANGE ME',
+    description: 'CHANGE ME',
+    author: 'commercetools',
+  },
+  plugins: [
+    {
+      resolve: '@commercetools-docs/gatsby-theme-docs',
+      options: {
+        websiteName: 'CHANGE ME',
```

Comment:
I would provide a proper name, like `Starter template` or something

Reply:
I have voluntarily chosen "CHANGE ME" in all places that have to be touched.   

I can't count the number of websites i've seen that still have the name of the template in the site title in production, so I think you must be able to simply search / replace  a certain term after using a starter and be sure to not look like an idiot with your site.

Reply:
Ok but the `websiteName` should not have whitespaces ideally.

Reply:
What I learnt here: I misunderstood  "websiteName" and was too lazy to look it up in the code of the theme.  I assumed (by variable name) that it's some sort of title and didn't even think of questioning why there already is a separate "title" metadata. 

I'll open a separate issue concerning whether websiteName could be automatically set to the path prefix.